src/react.ts

```diff
@@ -1,81 +1,95 @@
-import { useDebugValue } from 'react'
-import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/shim/with-selector'
 import createStore, {
-  EqualityChecker,
-  GetState,
-  SetState,
-  State,
-  StateCreator,
-  StateSelector,
-  StoreApi,
+  Store,
+  UnknownState,
+  StoreInitializer,
+  StoreMutatorIdentifier,
+  Mutate,
 } from './vanilla'
+import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/shim/with-selector'
+import { useDebugValue } from 'react'
+
+// ============================================================================
+// Types
+
+type UseStore = <S extends Store<UnknownState>, U = State<S>>(
+  store: S,
+  selector?: (state: State<S>) => U,
+  equals?: (a: U, b: U) => boolean
+) => U
+
+type Create = {
+  <
+    T extends UnknownState,
+    Mos extends [StoreMutatorIdentifier, unknown][] = []
```

Comment:
What's Mos mean?

Reply:
`MutationOuts`. The mutations that the initializer "outputs".

`M` is `Mutation`, `Mo` in `MutationOut`, `Mi` in `MutationIn`